freqtrade/rpc/api_server/api_ws.py

```diff
@@ -88,10 +89,8 @@ async def _process_consumer_request(
 
         # For every dataframe, send as a separate message
         for _, message in analyzed_df.items():
-            response = WSAnalyzedDFMessage(data=message)
-            await channel.send(response.dict(exclude_none=True))
-            # Throttle the messages to 50/s
-            await asyncio.sleep(0.02)
+            response = WSAnalyzedDFMessage(data=message).dict(exclude_none=True)
+            await channel_manager.send_direct(channel, response)
```

Comment:
Is there a downside to moving `.dict()` into the `send_direct` method? 

That'd obviously assume we can type-hint `send_direct()` with `WSMessageSchema` - but we shouldn't really be sending different messages anyway other than Ping anyway - so i think all messages SHOULD be typed already (i'm not sure about broadcast ...)

i don't really like the fact that "response" is now no longer typed - as it can introduce bad practices once we add additional message types

Reply:
I don't necessarily see a downside. I agree, I will make these changes.